servers/curated-corpus-api/schema-admin.graphql

```diff
@@ -696,6 +696,64 @@ input RemoveSectionItemInput {
   deactivateSource: ActivitySource
 }
 
+"""
+Input data for updating a Custom Editorial Section
+"""
+input UpdateCustomSectionInput {
+  """
+  An alternative primary key in UUID format supplied by ML.
+  """
+  externalId: ID!
+  """
+  The title of the custom section displayed to the users.
+  """
+  title: String
+  """
+  The description of the custom section displayed to the users.
+  """
+  description: String
+  """
+  An optional title used in hero modules.
+  """
+  heroTitle: String
+  """
+  An optional description or supporting text for use in hero modules.
+  """
+  heroDescription: String
+  """
+  The date of when the Section should go "live" for display on NewTab.
+  """
+  startDate: Date
+  """
+  An optional date of when the Section should stop being displayed.
+  """
+  endDate: Date
+  """
+  The GUID of the Scheduled Surface. Example: 'NEW_TAB_EN_US'.
+  """
+  scheduledSurfaceGuid: ID
+  """
+  Optional IAB metadata input
+  """
+  iab: IABMetadataInput
+  """
+  The source which created the Section.
+  """
+  createSource: ActivitySource!
```

Comment:
`createSource` should be replaced with `updateSource` here. sorry, I missed this in the tech spec, updated it: https://mozilla-hub.atlassian.net/wiki/spaces/FPS/pages/1467154643/Phase+3+Custom+Editorial+Sections+-+API+Technical+Specification+MVP#%E2%9C%8D%EF%B8%8F-Admin-GraphQL-Mutations-%26-Queries%3A